docs/integrations/databases/mongodb-atlas.md

```diff
@@ -323,18 +321,21 @@ In this section, you deploy the SAM application, which creates the necessary res
 
 The lambda function should be working now in sending logs to Sumo. You can check the CloudWatch logs in **Monitor** > **Logs** to see the logs of the function.
 
-#### Step 3B: Configure Collection for Multiple Projects (optional)
+13. Configuring collection for multiple projects (assuming you are already collecting Atlas data for one project). This task requires that you do the following:
```

Comment:
@himanshu219, this should be a new subsection. For example:

#### Configure collection for multiple projects

If you are already collecting Atlas data for one project, perform the following steps to configure for additional projects.

Reply:
sure let me create a separate subsection